src/har/page.cr

```diff
@@ -2,9 +2,21 @@ module HAR
   class Page
     include JSON::Serializable
 
+    # This module is used to skip ill-formed timestamps
+    module TimeConverter
+      def self.from_json(pull : JSON::PullParser)
+        Time::Format::ISO_8601_DATE_TIME.parse(pull.read_string)
+      rescue
```

Comment:
```suggestion
        Time::Format::ISO_8601_DATE_TIME.parse(pull.read_string) rescue nil
```

Reply:
Would it be worth logging the fail (sth along `Log.warn { "..." }`)?

Reply:
Indeed, logging would be great, but this kind of log lies deep and is only usable by the developers.
User can see Runtime Notifications, but it's going to be tricky to "catch" something from `har` to put it into Runtime Notification. I think, I can try to use custom log processor for har messages, this will also go to the following update.